Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -106,6 +106,15 @@
   #define IS_RRD_SC 1
   #define IS_U8GLIB_ST7920 1
 
+#elif ENABLED(SOVOL_SV06_RTS)
+  #ifndef LCD_SERIAL_PORT
+    #if MB(BTT_SKR_MINI_E3_V1_0, BTT_SKR_MINI_E3_V1_2, BTT_SKR_MINI_E3_V2_0, BTT_SKR_E3_TURBO)
+      #define LCD_SERIAL_PORT 1
+    #else
+      #define LCD_SERIAL_PORT 3 // Creality 4.x board
+    #endif
+  #endif
```

Comment:
This block choosing the `LCD_SERIAL_PORT` is out of date and is already handled in `Conditionals_adv.h` so it can be removed.